win32ss/user/rtl/image.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * PROJECT:   ReactOS RTL
+ * LICENSE:   LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:   RtlGetExpWinVer function
+ * COPYRIGHT: Copyright 2024 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
```

Comment:
Note that this is not the ReactOS RTL, but shared "run-time library" code between user32 and win32k only.
```suggestion
 * PROJECT:     ReactOS Win32ss RTL
 * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
 * PURPOSE:     RtlGetExpWinVer function
 * COPYRIGHT:   Copyright 2024 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
```
Extra spacing between the label and values, because values are aligned on 4-space-length multiples, following https://reactos.org/wiki/Coding_Style#File_Structure

Please also fix the copyright: you haven't written the function, it was added by commit https://github.com/reactos/reactos/commit/1f78e8ec47a34869d37f05d03d6fca16707e70fc#diff-01e2f4fbe7e6c65e825aee29fe7041bdb8afaf8927ff1f0b6334153479130214R151 ,
but you moved it from user32 and slightly adapted it. So a `Copyright 2019 James Tabor <james.tabor@reactos.org>` line needs to be added.

Reply:
James Tabor added.